### DISCUSSION TITLE
New Note Types

### DISCUSSION THREAD

**Discussion by Onemario1234** [Leather128/LeathersFunkinEngine#41] [Ideas]
Here are some suggestions for new note types I thought up on a whim.

Damage Notes: Self-explanatory. They damage you and count as a miss if you hit them.

Healing Notes: The opposite of Damage Notes. Hitting them heals a large chunk of your health.

Opponent Damage Notes: Used entirely by your opponent, these notes could have your opponents damage your health when hit, the amount of damage dealt being the same damage BF deals to the opponent when hitting a note. That kind of thing.

Opponent Miss Notes: When applied to the opponent's side, they completely miss this note. It scrolls by the arrows and they don't play an animation. Whether it heals you a bit upon them missing it is another story, but it could be interesting.

That's all I can think of off the top of my head. Thoughts?

**Comment by Leather128** [comment]
sounds cool, might make some of these
